awx/ui/client/src/templates/prompt/steps/preview/prompt-preview.partial.html

```diff
@@ -0,0 +1,84 @@
+        <div class="Prompt-previewRowValue" ng-bind="promptData.prompts.inventory.value.name"></div>
+    </div>
+    <div class="Prompt-previewRow--flex">
+        <div class="Prompt-previewRowTitle">{{:: vm.strings.get('prompt.LIMIT') }}</div>
+        <div class="Prompt-previewRowValue" ng-bind="promptData.prompts.limit.value"></div>
+    </div>
+    <div class="Prompt-previewRow--flex">
+        <div class="Prompt-previewRowTitle">{{:: vm.strings.get('prompt.VERBOSITY') }}</div>
+        <div class="Prompt-previewRowValue" ng-bind="promptData.prompts.verbosity.value.label"></div>
+    </div>
+    <div class="Prompt-previewRow--noflex">
+        <div class="Prompt-previewRowTitle">
+            <span>{{:: vm.strings.get('prompt.JOB_TAGS') }}&nbsp;</span>
+            <span ng-click="vm.showJobTags = !vm.showJobTags">
+                <span class="fa fa-caret-down" ng-show="vm.showJobTags" ></span>
+                <span class="fa fa-caret-left" ng-show="!vm.showJobTags"></span>
+            </span>
+        </div>
+        <div ng-show="vm.showJobTags" style="display: flex">
+            <div class="Prompt-previewTagContainer u-wordwrap" ng-repeat="tag in promptData.prompts.tags.value">
+                <div class="LabelList-tag">
+                    <span>{{tag.name}}</span>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="Prompt-previewRow--noflex">
+        <div class="Prompt-previewRowTitle">
+            <span>{{:: vm.strings.get('prompt.SKIP_TAGS') }}&nbsp;</span>
+            <span ng-click="vm.showSkipTags = !vm.showSkipTags">
+                <span class="fa fa-caret-down" ng-show="vm.showSkipTags" ></span>
+                <span class="fa fa-caret-left" ng-show="!vm.showSkipTags"></span>
+            </span>
+        </div>
+        <div ng-show="vm.showSkipTags" style="display: flex">
+            <div class="Prompt-previewTagContainer u-wordwrap" ng-repeat="tag in promptData.prompts.skipTags.value">
+                <div class="LabelList-tag">
+                    <span>{{tag.name}}</span>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="Prompt-previewRow--flex">
+        <div class="Prompt-previewRowTitle">{{:: vm.strings.get('prompt.SHOW_CHANGES') }}</div>
+        <div class="Prompt-previewRowValue">
+            <span ng-if="promptData.prompts.diffMode.value">{{:: vm.strings.get('ON') }}</span>
+            <span ng-if="!promptData.prompts.diffMode.value">{{:: vm.strings.get('OFF') }}</span>
+        </div>
+    </div>
+    <div class="Prompt-previewRow--noflex">
+        <div class="Prompt-previewRowTitle">{{:: vm.strings.get('prompt.EXTRA_VARIABLES') }}</div>
+        <div>
+            <textarea rows="6" ng-model="promptExtraVars" name="preview_variables" class="form-control Form-textArea Form-textAreaLabel" id="job_launch_preview_variables"></textarea>
```

Comment:
It might be good to make the cursor disabled on this readonly CodeMirror instance.